docs/mastering-plone-5/instructions_plone5/index.md

```diff
@@ -0,0 +1,382 @@
+
+[Vagrant](https://www.vagrantup.com) is a tool for building complete development environments.
+
+We use it together with Oracle’s [VirtualBox](https://www.virtualbox.org) to create and manage a virtual environment.
+
+(plone5-install-virtualbox)=
+
+### Install VirtualBox
+
+Vagrant uses Oracle’s VirtualBox to create virtual environments.
+
+Here is a link directly to the download page: <https://www.virtualbox.org/wiki/Downloads>.
+
+We use VirtualBox 6.0.x
+
+(plone5-instructions-configure-vagrant-label)=
+
+### Install and configure Vagrant
+
+Get the latest version from <https://developer.hashicorp.com/vagrant/install> for your operating system and install it.
+
+Now your system has a command {command}`vagrant` that you can run in the terminal.
+
+First, create a directory in which you want to do the training.
+
+```{warning}
+If you already have a {file}`training` directory because you followed the **Installing Plone without vagrant** instructions above,
+you should either delete it, rename it, or use a different name below.
+```
+
+```shell
+mkdir training
+cd training
+```
+
+Setup Vagrant to automatically install the current guest additions.
+You can choose to skip this step if you encounter any problems with it.
+
+```shell
+vagrant plugin install vagrant-vbguest
+```
+
+Now download {download}`plone_training_config.zip <../../_static/plone_training_config.zip>` and copy its contents into your training directory.
+
+```shell
+wget https://github.com/plone/training/raw/refs/heads/main/docs/_static/plone_training_config.zip
+unzip plone_training_config.zip
+```
+
+The training directory should now hold the file {file}`Vagrantfile` and the directory {file}`manifests` which again contains several files.
+
+Now start setting up the virtual machine (VM) that is configured in {file}`Vagrantfile`:
+
+```shell
+vagrant up
+```
+
+This takes a **veeeeery loooong time** (between 10 minutes and 1h depending on your Internet connection and system speed) since it does all the following steps:
+
+- downloads a virtual machine (Official Ubuntu Server 18.04 LTS, also called "Bionic Beaver")
+- sets up the VM
+- updates the VM
+- installs various system-packages needed for Plone development
+- clones the training buildout into /vagrant/buildout
+- builds Plone annd installs all dependencies
+
+````{note}
+Sometimes this stops with the message:
+
+```shell
+Skipping because of failed dependencies
+```
+````
+
+If this happens or you have the feeling that something has gone wrong and the installation has not finished correctly for some reason
+you need to run the following command to repeat the process.
+
+This will only repeat steps that have not finished correctly.
+
+```shell
+vagrant provision
+```
+
+You can do this multiple times to fix problems, e.g. if your network connection was down and steps could not finish because of this.
```

Comment:
```suggestion
You can do this multiple times to fix problems, for example, if your network connection was down and steps could not finish because of this.
```